scene/gui/color_mode.cpp

```diff
@@ -245,8 +245,6 @@ bool ColorModeRAW::apply_theme() const {
 		slider->remove_theme_icon_override("grabber");
 		slider->remove_theme_icon_override("grabber_highlight");
 		slider->remove_theme_style_override("slider");
```

Comment:
```suggestion
		slider->remove_theme_constant_override("grabber_offset");
```
The follow-up of using `grabber_offset` instead of `slider`.